src/Components/Home.js

```diff
@@ -63,85 +69,73 @@ function HomePage() {
-          areas, including professional development, resume building, and mentor
-          circles
-        </p>
-        <h3 className="meetingTime">Meeting Times/Location (Spring 2024)</h3>
-        <p className="meetingDetails">
-          Every week on Wednesday, from 5:30-6:30pm
-          <br />
-          Location: Loew 216
-        </p>
-      </div>
-      <div className="map-and-label">
-        <img className="map" src={SWECCmap} alt="map" />
-        <p className="mapLabel">Location: Loew 216</p>
-      </div>
-    </div>
-  );
-
-  const posts = (
-    <div className="posts">
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/C2-o7roPbfN"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/C23PyAWPYdW"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/CyRondZPVVS"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/Cxum20yOlkQ"
-      />
-      <InstagramEmbed
-        className="instaPost slide-up"
-        url="https://www.instagram.com/p/CxrxWcQLpSb"
+  function intPostsWidget() {
+    return (
+      <BeholdWidget
+        onLoad={() => console.log("Loaded!")}
+        feedId="5rAX7PhyjFjmyVfW4Plm"
       />
-    </div>
-  );
+    );
+  }
 
   return (
     <div>
       <Carousel />
 
-      <div className="whatWeDoSection">
-        <h2 className="summaryTitle">Who we are</h2>
-        <p className="summary">
-          The Software Engineering Career Club (SWECC) is a student-led
-          organization at the University of Washington in Seattle. Our mission
-          is to support and empower individuals interested in pursuing a
-          successful career in software engineering. We offer a wide range of
-          activities and resources, including networking events, LeetCode
-          challenges, mentorship programs, and interview preparation, to help
-          our members thrive in the field. Join SWECC to jumpstart your software
-          engineering career with confidence and connections!
+      <div className="mainPage-body">
+        <TextLeftImageRight
+          summary="The Software Engineering Career Club (SWECC) is a student-led
+organization at the University of Washington in Seattle. Our mission
+is to support and empower individuals interested in pursuing a
+career in software engineering. We offer a wide range of
+activities and resources, including networking events, resume reviews,
+mentorship programs, interview preparation, and more. Join us to jumpstart your software engineering career!"
+          summaryTitle="Who we are"
+          image={img3}
+        />
+        <ColFlexDoubleTextRightImageLeft
+          title1="SWECC Meetings"
+          title2="Meeting Times/Location (Autumn 2024)"
```

Comment:
@elimelt we should think about adding to our backend and admin dashboard to allow this shit to be edited and read from the server. We can also add those instagram things to there. 

What do you think? I might work on this over the break.

Reply:
Yeah something like your personal site would be useful, but it can be super simple